tools/build_defs/repo/git.bzl

```diff
@@ -99,11 +99,11 @@ _common_attrs = {
     "patches": attr.label_list(
         default = [],
         doc =
-            "A list of files that are to be applied as patches afer " +
+            "A list of files that are to be applied as patches after " +
             "extracting the archive.",
     ),
     "patch_tool": attr.string(
-        default = "patch",
+        default = "",
```

Comment:
Why should we remove default?

Reply:
I see, so it is a way to specify "force fallback to patch tool" - but in the example above it is that there is a test for fallback to patch tool, and patch tool is not specified... what is the algorithm then?

Reply:
I'll add document about this in later PR when I modify `git.bzl` and `http.bzl` ;)

Reply:
I guess there should be an answer about the algorithm when the reviewer asks you that :)

Reply:
Oh, sorry, the algorithm is to fallback to patch command line tool when `patch_tool` is specified or there are arguments other than `-p` in `patch_args`.